cli/JSXV4.md

```diff
@@ -29,6 +29,8 @@ To build certain dependencies in V3 compatibility mode, whatever the version use
 
 In V3 compatibility mode, the listed dependencies are built in V3 mode, and in addition `-open ReatcV3` is added to the compiler options, so that the `ReactV3` compatibility module in rescript-react is used.
 
+> Note that `@rescript/react` is not necessary adding in the `v3-dependencies`, or it would make the cyclic dependencies error.
```

Comment:
Note: do not add `@rescript/react`in the `v3-dependencies`, or it will cause a cyclic dependencies error.

Reply:
Thanks! https://github.com/rescript-lang/syntax/pull/659/commits/eb89eb05271d35df9feaaf50a2e3eea11b58a827